packages/template-retail-react-app/app/components/store-locator/list.jsx

```diff
@@ -23,6 +24,7 @@ export const StoreLocatorList = () => {
     const [initialSelectedStoreId, setInitialSelectedStoreId] = useState(selectedStoreId)
 
     const hasItemsInBasket = derivedData?.totalItems > 0
+    const storeSelectionDisabled = MULTISHIP_IS_ENABLED ? false : hasItemsInBasket
```

Comment:
What does this mean? How does store selection relate to multi ship and basket items?

Reply:
@alexvuong For BOPIS without multiship the store selection cannot be changed unless the basket is empty (existing BOPIS behavior).  With multiship this restriction is removed.

Reply:
gotcha, there is more than what meets the eye with just code, let's add some context and rename to make it clear
``
```suggestion. 
    //  with BOPIS enabled: store locator selector can't be changed unless basket is emty
    // with BOPIS and MULTI_SHIP enabled, store locator can be changed any time
    const shouldStoreSelectionDisabled = MULTISHIP_IS_ENABLED ? false : hasItemsInBasket
``

Reply:
If my member serves my well, this component will not render unless BOPIS is enabled, correct?

Reply:
`StoreLocatorModal` is removed by the `STORE_LOCATOR_IS_ENABLED===false` in `_app/index.js`

Reply:
adding this suggested comment with W-19168602